carma_guidance_plugins/src/plugin_base_node.cpp

```diff
@@ -142,7 +142,7 @@ namespace carma_guidance_plugins
     }
 
     carma_planning_msgs::msg::Plugin msg;
-    msg.name = get_plugin_name();
+    msg.name = std::string(get_namespace()) + std::string(get_name());
```

Comment:
Are we using the fully qualified names here? I believe previously we had been only using the name of the node/plugin, but no namespace information as part of the plugin contract is that their service calls can be accessed by /guidance/plugins/PLUGIN_NAME/plan_maneuvers, etc. 

Not a big deal either way, but worth a quick discussion of trade-offs i think.

Reply:
Actually, I've changed my mind on this. While I would prefer to use the fully qualified name, it seems that a large amount of changes would be needed in the ros1 nodes to remain compatible. So instead, we'll go with the base name of the node and used the /guidance/plugins prefix